punishments.js

```diff
@@ -593,7 +593,7 @@ Punishments.roomPunish = function (room, user, punishment, recursionKeys) {
 			rest: rest,
 		}, roomid + ':' + id, ROOM_PUNISHMENT_FILE);
 
-		if (roomid === "battle" || !(room.isPrivate === true || room.isPersonal || room.battle)) Punishments.monitorRoomPunishments(user);
+		if (roomid === 'string' || !(room.isPrivate === true || room.isPersonal || room.battle)) Punishments.monitorRoomPunishments(user);
```

Comment:
Im pretty sure this is supposed to be `typeof roomid === 'string'`

Reply:
Whoops still need to do the “your” grammar fix too